fern/api/definition/apiLoad.yml

```diff
@@ -0,0 +1,38 @@
+imports:
+  commons: commons.yml
+types:
+  Api:
+    properties:
+      apiId: commons.ApiId
+      packages: list<Package>
+  Package:
+    properties:
+      packageId: commons.PackageId
+      name: string
+      endpoints: list<Endpoint>
+  Endpoint:
+    properties:
+      endpointId: commons.EndpointId
+      endpointName: string
+  Type:
+    properties:
+      typeId: commons.TypeId
+      typeName: string
+  Error:
+    properties:
+      errorId: commons.ErrorId
+      errorName: string
+      statusCode: string
+
+services:
+  http:
+    ApiLoadService:
+      auth: false
+      base-path: /{apiId}
+      path-parameters:
+        apiId: commons.ApiId
```

Comment:
locally the api id just has to be the `api name`, in the cloud itll need to be `org id + api id`. We can ditch figuring this out now but just something i thought of when i saw this

Reply:
let's discuss in person